app/services/articles/feeds/weighted_query_strategy.rb

```diff
@@ -119,7 +119,7 @@ class WeightedQueryStrategy
         # Weight to give to the number of comments on the article.
         comments_count_factor: {
           clause: "articles.comments_count",
-          cases: [[0, 0.9], [1, 0.92], [2, 0.94], [3, 0.96], [4, 0.98]],
+          cases: (0..9).map { |n| [n, 0.8 + (n / 50.0)] },
```

Comment:
Wouldn't this be clearer?

```suggestion
          cases: (0..9).map { |n| [n, 0.8 + 0.02 * n] },
```